SECURITY_CONTACTS

```diff
@@ -0,0 +1,11 @@
+# Reporting a security vulnerability
+
+Please do not disclose any security issue publicly e.g. Pull Requests, Issues, Comments.
+
+In this repository security reports are handled according to the
+Metal3-io project's security policy. For more information about the security
+policy consult the User-Guide [here](https://metal3io.netlify.app/security_policy.html).
+
+As this repository has a rolling release cycle and there are no release branches the security
+fix will be merged to the main branch and depending on the severity of the vulnerability the issue
+will either got it's own patch release or it will be released together with the next upcoming regular release.
```

Comment:
too complex wording + a typo or two, so maybe something like:

will either be released in a patch release or in the next upcoming regular release.

Reply:
Done.